src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,100 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final double DEFAULT_COEFFICIENT = 1.5;
+    private static final String INVALID_INDEX_MSG = "The index passed to the method is invalid";
+    private int defaultLength = 10;
```

Comment:
It can be constant too

Reply:
I change this variable in the method resizing()...